fontforge/sfd.c

```diff
@@ -2813,17 +2816,19 @@ static int SFD_Dump( FILE *sfd, SplineFont *sf, EncMap *map, EncMap *normal,
 	for ( remap = map->remap; remap->infont!=-1; ++remap )
 	    fprintf(sfd, "Remap: %x %x %d\n", (int) remap->firstenc, (int) remap->lastenc, (int) remap->infont );
     }
-    if ( sf->display_size!=0 )
+    if ( sf->display_size!=0 && SaveEditorState )
 	fprintf( sfd, "DisplaySize: %d\n", sf->display_size );
     if ( sf->display_layer!=ly_fore )
```

Comment:
how is this not part of editor state?

Reply:
IIRC, this affects the exported font. The display layer is the layer that will be used for exported outlines.